src/caw/functions/caw_functions.cpp

```diff
@@ -0,0 +1,246 @@
+// This is a file that will contain the caw functions
+// to be registered with, and used by Faz
+#include "caw_functions.h"
+
+#include <chrono>
+#include <ctime>
+#include <sstream>
+#include <string>
+#include <sys/time.h>
+#include <vector>
+
+bool isInList(const std::vector<std::string> &list, const std::string &key) {
+  for (std::string s : list) {
+    if (s == key) {
+      return true;
+    }
+  }
+  return false;
+}
+
+std::vector<std::string> stringToVector(const std::string &strToConvert) {
```

Comment:
See https://developers.google.com/protocol-buffers/docs/reference/cpp-generated.

Reply:
Hmm initially I was using this to separate caw children (stored in a string as "1 2 3" etc. But that was because I was separating and storing caw's in individual vectors. This way of storing it is much better. Have changed the implementation!